sips/cluster_consensus.md

```diff
@@ -0,0 +1,127 @@
+|     Author     |           Title            |  Category  |       Status        |    Date    |
+| -------------- | -------------------------- | ---------- | ------------------- | ---------- |
+| Matheus Franco | Cluster consensus          | Core       | open-for-discussion | 2024-03-05 |
+
+## Summary
+
+Aggregate `Attestation` and `Sync Committee` duties based on the cluster of operators and the duties' slot. Also, merge the post-consensus partial signature messages for such duties into a single message.
```

Comment:
I don't see in the SIP nothing about merging post consensus?

Reply:
Added more detail